impress-org/givewp
# Fix: display errors on Multi-Step Forms

Resolves #6670

## Description



The errors thrown by the backend weren't getting displayed on forms that use the Multi-step form template because the frontend function that handles the server errors was using elements that only exist on the classic and legacy form templates.

This PR changes it and makes the function also checks for an element that exists only on the Multi-step form template and appends the errors in the proper place when the backend sends them.

## Affects



Multi-Step Forms.

## Visuals



![image](https://user-images.githubusercontent.com/16308864/212981256-e8c17d17-87c1-445a-8028-8b41d6094630.png)

## Testing Instructions



1. Create a donation with a new user/email - after the donation, you'll get logged in with the new user automatically.
2. Now open a private tab in your browser and try to do a new donation with the same user/email create previously.
3. The form scrolls to the top and the following error is displayed: `The email address provided is already in use by a registered account on this site. Please log in and try again.`

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203767103828497
